gpt4all-chat/modellist.h

```diff
@@ -180,25 +186,27 @@ struct ModelInfo {
     QString m_url;
     QString m_quant;
     QString m_type;
-    bool    m_isClone              = false;
-    bool    m_isDiscovered         = false;
-    int     m_likes                = -1;
-    int     m_downloads            = -1;
+    bool    m_isClone                 = false;
+    bool    m_isDiscovered            = false;
+    int     m_likes                   = -1;
+    int     m_downloads               = -1;
     QDateTime m_recency;
-    double  m_temperature          = 0.7;
-    double  m_topP                 = 0.4;
-    double  m_minP                 = 0.0;
-    int     m_topK                 = 40;
-    int     m_maxLength            = 4096;
-    int     m_promptBatchSize      = 128;
-    int     m_contextLength        = 2048;
-    mutable int m_maxContextLength = -1;
-    int     m_gpuLayers            = 100;
-    mutable int m_maxGpuLayers     = -1;
-    double  m_repeatPenalty        = 1.18;
-    int     m_repeatPenaltyTokens  = 64;
-    QString m_promptTemplate       = "### Human:\n%1\n\n### Assistant:\n";
-    QString m_systemPrompt         = "### System:\nYou are an AI assistant who gives a quality response to whatever humans ask of you.\n\n";
+    double  m_temperature             = 0.7;
+    double  m_topP                    = 0.4;
+    double  m_minP                    = 0.0;
+    int     m_topK                    = 40;
+    int     m_maxLength               = 4096;
+    int     m_promptBatchSize         = 128;
+    int     m_contextLength           = 2048;
+    mutable int m_maxContextLength    = -1;
+    int     m_gpuLayers               = 100;
+    mutable int m_maxGpuLayers        = -1;
+    double  m_repeatPenalty           = 1.18;
+    int     m_repeatPenaltyTokens     = 64;
+    QString m_promptTemplate          = QObject::tr("### Human:\n%1\n\n### Assistant:\n");
+    QString m_systemPrompt            = QObject::tr("### System:\nYou are an AI assistant who gives a quality response to whatever humans ask of you.\n\n");
+    QString m_chatNamePrompt          = QObject::tr("Describe the above conversation in seven words or less.");
+    QString m_suggestedFollowUpPrompt = QObject::tr("Suggest three very short factual follow-up questions that have not been answered yet or cannot be found inspired by the previous conversation and excerpts.");
```

Comment:
I can see only problems with the UI language setting being directly connected to what language the LLM is using. If we are to translate these strings, this should absolutely be done separately&mdash;I don't think the Germans on our Discord server would be happy if Llama 3 suddenly stopped speaking English.